app/concerns/liquid_droppable.rb

```diff
@@ -82,9 +84,49 @@ def to_liquid
     end
   end
 
+  class ActiveRecordCollectionDrop < Drop
+    include Enumerable
```

Comment:
What functionality of `Enumerable` is used here?

Reply:
Most standard filters that take an array expect its parameter to be an Enumerable if it's not an Array: https://github.com/Shopify/liquid/blob/master/lib/liquid/standardfilters.rb (grep for `InputIterator`)

Reply:
Right, I missed to point of having `Enumerable` to give us all it's functionally based on `each`.